pkg/pool-manager/pool_test.go

```diff
@@ -86,6 +114,17 @@ var _ = Describe("Pool", func() {
 			_, err = NewPoolManager(fakeClient, startPoolRangeEnv, endPoolRangeEnv, false)
 			Expect(err).To(HaveOccurred())
 		})
+
+		It("should fail to pool manager because of the first octet of RangeEnd is not 02, 06, 0A, 0E", func() {
```

Comment:
should fail to create a pool manager object when the first octet of RangeEnd is not 02, 06, 0A, 0E